utils/permissions/test/permissions.test.mjs

```diff
@@ -0,0 +1,167 @@
+import {
+  check,
+  request,
+  checkNotifications as rnpCheckNotifications,
+  requestNotifications as rnpRequestNotifications,
+} from 'react-native-permissions'
+import { is } from '@ma-shop/is'
+
+import { permissions } from '../src'
+
+
+jest.mock('react-native-permissions', () => ({
+  check: jest.fn().mockResolvedValue('granted'),
+  checkNotifications: jest.fn().mockResolvedValue({ status: 'granted' }),
+  PERMISSIONS: {
+    ANDROID: {
+      ACTIVITY_RECOGNITION: 'motion-android',
+      ACCESS_FINE_LOCATION: 'locationAlways-android',
+    },
+    IOS: {
+      MOTION: 'motion-ios',
+      LOCATION_ALWAYS: 'locationAlways-ios',
+      LOCATION_WHEN_IN_USE: 'locationWhenInUse-ios',
+    },
+  },
+  request: jest.fn().mockResolvedValue('granted'),
+  requestNotifications: jest.fn().mockResolvedValue({ status: 'granted' }),
+  RESULTS: { GRANTED: 'granted' },
+}))
+
+
+describe('permissions', () => {
+  test('requestNotifications', async () => {
+    await permissions.requestNotifications()
+    expect(rnpRequestNotifications).toBeCalledWith([ 'alert' ])
+  })
+
+  test('checkNotifications', async () => {
+    const status = 'denied'
+
+    is.ios = jest.fn(() => false)
```

Comment:
Would it make sense to add a `beforeEach` block to reset `is.ios()` to be either true or false? You'd be able to eliminate half of the times you set it in this file